modules/SofaGeneralAnimationLoop/MechanicalMatrixMapper.inl

```diff
@@ -138,6 +139,19 @@ void MechanicalMatrixMapper<DataTypes1, DataTypes2>::parseNode(sofa::simulation:
             }
         }
     }
+    for(unsigned int i=0; i<node->interactionForceField.size(); i++)
```

Comment:
no need to change the iterator if I well remember. Except if you assumer your container will go other the unsigned int limit and reach the long long limit.